### ISSUE TITLE
Bungomachine broke, No bungotoxin for you.

### ISSUE THREAD

**Issue by xXPawnStarrXx** [Skyrat-SS13/Skyrat-tg#3026]
Reporting client version: 513.1530

## Round ID:
656

## Testmerges:
- [Lopland Security and Armadyne Weaponry rework [Security Medic and Sergeant] [RFT]](https://github.com/Skyrat-SS13/Skyrat-tg/pull/2860)
- [[NON-MODULAR] Porting audible emotes from oldbase.](https://github.com/Skyrat-SS13/Skyrat-tg/pull/2795)
- [[Modular] Aphrodisiacs](https://github.com/Skyrat-SS13/Skyrat-tg/pull/2914)
- [[Semi-modular] View Latest Ticket ahelp command for players](https://github.com/Skyrat-SS13/Skyrat-tg/pull/2958)

## Reproduction:

When growing bungo trees, you can harvest pits from them. The trees produce fruit consistantly, but upon collecting nine pits, you cannot produce any more pits. None of the methods; Eating, activating or grinding the fruit yeilds any more pits, I have recreated this issue multiple times now to the safe effect. Limiting the total bungotoxin production at 69.3u meaning you cannot make mourning poultice enmass.

**Comment by Quill-Weave** [comment]
![image](https://user-images.githubusercontent.com/103809013/163684173-0e1d825f-dc4c-45b2-a025-9038155138c9.png)

Tested! Bungo Fruits work in a interesting fashion. 

When the Bungo Fruit is used in hand, the old object (in this case, the fruit!) is deleted, and the pit is spawned.

When it runs out of nutriment and such when you eat it, the code works the same way, to my knowledge. Old object removed, pit spawns.

But when you grind a Bungo Fruit down, the fruit is turned directly into Enzyme and Nutriment. This has the side-effect of not spawning a pit due to the original object being ground up. I went and tested it in a mortar and pestle, just to be sure. It juices the fruit, and when you juice it in the grinder, it has the same effect! No pit! 

I'm inclined to think that it's how the fruit was originally coded, but we'll have to check in with /tg/ to be sure. 

TL;DR: Cannot reproduce!
